src/pages/help/faq.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: FAQ
+keywords: 'IBM,brand,faq'
```

Comment:
We should probably add a `description:` field here also, thats what shows up when someone uses the search.

Reply:
added description..